components/_impl/workers/subprocess_rpc.py

```diff
@@ -229,6 +250,14 @@ def write(self, msg: bytes) -> None:
 
         os.write(self.write_fd, packed_msg)
 
+    def get_writer_pid(self) -> int:
+        assert self._writer_pid, "Writer pid is not specified. Maybe calling from child process or input pipe.\
```

Comment:
nit: `assert self._writer_pid is not None` is clearer, since zero is falsy.

Reply:
Aha, thanks for catching that!